.travis.yml

```diff
@@ -48,12 +43,12 @@ env:
     - TEST_SUITE=pdk
 
 install:
+  - make setup-ci
   - source .ci/setup_env.sh
```

Comment:
We need to run the make task as well as `source` the bash script so the bash script can export the necessary environment variables which isn't possible via Make